packages/api/src/router/user.ts

```diff
@@ -3,19 +3,17 @@ import { z } from "zod";
 
 import { can, generateApiKey, hash } from "@ctrlplane/auth/utils";
 import { and, eq, takeFirst } from "@ctrlplane/db";
-import { scopeType, user, userApiKey } from "@ctrlplane/db/schema";
+import { createUser, scopeType, user, userApiKey } from "@ctrlplane/db/schema";
 
 import { createTRPCRouter, protectedProcedure } from "../trpc";
 
 export const profileRouter = createTRPCRouter({
   update: protectedProcedure
-    .input(
-      z.object({ activeWorkspaceId: z.string().uuid().nullable().optional() }),
-    )
+    .input(createUser)
     .mutation(({ ctx, input }) =>
       ctx.db
         .update(user)
-        .set(input)
+        .set(createUser.parse(input))
```

Comment:
```suggestion
        .set(input)
```

trpc already parses this